web-api/openapi/front_office.yaml

```diff
@@ -65,7 +65,7 @@ paths:
       summary: Add interaction between a user and a document
       description: |-
         The interaction is used to provide personalized documents to the user.
-      operationId: documentInteraction
+      operationId: updateUserInteractions
```

Comment:
Should this not be createUserInteraction?
As if you do 2 positive interactions, it just runs twice on the CoI level, right?